accepted/2020/minimum-os-version/minimum-os-version.md

```diff
@@ -0,0 +1,246 @@
+
+However, for libraries that are distributed as NuGets, it is useful to be able to multi-target across multiple API versions. This allows NuGets to access features available in newer API versions while continuing to ship updates for consumption by apps and libraries that (for whatever reason) target older API versions.
+
+By making the OS API version part of the target framework moniker (TFM), we take advantage of all the existing TFM-based mechanisms for multi-targeting builds against multiple versions of the API, selecting multi-targeted builds of a library from NuGets, and validating binary compatibility of references.
+
+The OS API version has major, minor, servicing and binding revision components. This takes the following form:
+
+```
+{major}[.{minor}[.{servicing}]][-r{bindingRevision}]
+```
+
+ When a component is not specified, this is equivalent to it being zero. The major, minor and servicing components of the version should match the OS version. The binding revision may be incremented if the OS binding is updated independently of OS updates. For example, if `net5.0-ios15.0` is released with some missing bindings, it could be followed by a release `net5.0-ios15.0-r1` that adds them.
+
+## OS Target Version
+
+The OS Target Version is used on certain platforms to enable backwards compatibility behaviors so that old apps continue to run correctly when API behaviors change in newer OS versions. For example, when an app that targeted Android 9.0 was run on Android 10.0, the OS might “quirk” the API behavior to continue to match the 9.0 behavior, but apps that targeted 10.0 or newer would get the newer API behavior.
+
+For now, handling of this version is left out of the scope of this document. Although it would be good to have a standard pattern in how this is defined across platforms, it’s not a universal concept, and doesn’t easily map into useful tooling. It’s also an app-level setting, and not useful for libraries except perhaps for warning of potential incompatibility, but that would have a very high false positive rate.
+
+## OS Minimum Version
+
+The OS Minimum Version is the minimum version of the OS that an app or library supports running on. If it is not specified, it should default to the OS API Version. If an app or library author specifies an OS Minimum Version, they are declaring that they are taking responsibility for ensuring that the app can run on that version and higher by using runtime checks to guard calls to potentially unavailable APIs. This allows apps and libraries to progressively “light up” features on newer OS versions while continuing to run downlevel.
+
+We will make this concept first class and propagate the value throughout the build system and NuGet consistently, which will enable us to build analyzers and other experiences to help developers follow this pattern. This tooling will be detailed later in this document.
+
+# API Annotations
```

Comment:
It might be helpful to outline our guidelines for 3rd party TFM libraries. 
- should they always have the annotation
- or only when they directly call to platform api
- or only when they wrap annotated api
- etc

Reply:
Added some very basic guidance on this